### PR TITLE
Updates rails to ~> 7.0.6

### PR DESCRIPTION
**Pull request by ottony**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 ruby "3.1.2"
 
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
-gem "rails", "~> 7.0.2"
+gem "rails", "~> 7.0.6"
```
Comment (julianalucena) [inline Gemfile:7]: What do you think of removing the minor number? This way, dependabot could update by itself.

**Review by julianalucena — APPROVED** [review]

File: Gemfile (hunk 1 above)
Comment (ottony) [reply Gemfile:7]: Nice idea.

You mean `'~> 7.0'`, right?

Comment (ottony) [reply Gemfile:7]: With `'>= 7.0.6'` dependabot will continue having problems updating.

But without them, maybe it allows a machine with rails `7.0.1` to run for example.
image
 
I'll read about it.

Comment (ottony) [reply Gemfile]: Done

> But without them, maybe it allows a machine with rails 7.0.1 to run for example.

I've made some tests, it's not true

**Pull request merged**